styles/trinket/catppuccin.user.css

```diff
@@ -593,14 +676,22 @@
           }
         }
 
-        &.danger,
-        &.caution {
+        &.danger {
           &,
           &:hover {
             background-color: @red !important;
+            border-color: @red !important;
             color: @base !important;
```

Comment:
Hmm surely we should have some hover state here? Like the background shouldn't be the same when hovered vs not hovered?

Reply:
but what would it be? red button become base? that would be counter intuitive imo

Reply:
oh yea makes sense